Add New Relic enable|disable commands

rvtraveller opened the pull request:
Adds commands for enabling/disabling New Relic which follows the structure of enabling/disabling Solr.  I pulled the workflows from what I could find in the Dashboard Javascript so I'm hoping the workflows are the right ones to use in Terminus..  I also added some test features for the new commands but was unable to test the tests.  I can confirm that the actual commands in Terminus work for me.

Reference:  #102

Commit:
Add command for enabling/disabling New Relic Pro

Commit:
Add test scenarios for enabling/disabling New Relic.

Commit:
Add @return documentation to New Relic Pro functions in Site model.

Commit:
Minor code style cleanup to keep the tests happy

SaraMcCutcheon commented:
This is a great contribution, thanks @rvtraveller. Can we make these both subcommands of `site new-relic`, though? Having one named `site new-relic` and another being `site new-relic-pro` is redundant.

Commit:
Combine New Relic commands to remove redundancies.

rvtraveller committed:
Merge branch 'master' into new-relic-integration

rvtraveller commented:
@TeslaDethray makes sense.  I was a little hesitant to combine them originally because I wasn't sure of any changes happening around New Relic with the announcement yesterday.  I updated the PR to have them combined and also fixed an issue with the old New Relic command always reporting that New  Relic was disabled (we were checking the wrong key in `$data`).
